core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5346,6 +5346,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey FUSE_TMP_FOLDER =
+      new Builder(Name.FUSE_TMP_FOLDER)
+          .setDefaultValue("")
```

Comment:
we can remove this line, i remember there is a Configuration.isSet function to checking whether a variable is set

Reply:
prefer not depending on `Configuration.isSet` to ease maintaining